homeassistant/components/humidifier/__init__.py

```diff
@@ -192,6 +198,14 @@ def available_modes(self) -> list[str] | None:
         """
         return self._attr_available_modes
 
+    @property
+    def operating_state(self) -> str | None:
```

Comment:
This requires an architecture discussion.

https://developers.home-assistant.io/docs/core/entity#changing-the-entity-model

For background, here's the discussion issue that was had before the integration was created:
https://github.com/home-assistant/architecture/issues/310

Reply:
Opened https://github.com/home-assistant/architecture/discussions/731

Reply:
What can I do to get the architecture proposal under more people's eyes?

Reply:
There needs to be an interest from the member community to drive an architecture discussion forward. Not much to do, besides waiting.